include/vrv/object.h

```diff
@@ -379,6 +379,13 @@ class Object : public BoundingBox {
     void FindAllChildByAttComparison(ArrayOfObjects *objects, AttComparison *attComparison,
         int deepness = UNLIMITED_DEPTH, bool direction = FORWARD, bool clear = true);
 
+    /**
```

Comment:
Same here: with FindAllChildByAttComparison will work with a class inheriting from AttComparison

Reply:
Do you mean that FindAllChildByInterfaceComparison will work?

Reply:
The method should be:
```
void FindAllChildByComparison(ArrayOfObjects *objects, Comparison *comparison,
         int deepness = UNLIMITED_DEPTH, bool direction = FORWARD, bool clear = true);
```
And then we can call it with either an `AttComparison` or and `InterfaceComparison`.